truss/truss/config.go

```diff
@@ -0,0 +1,41 @@
+package truss
+
+import (
+	"path/filepath"
+)
+
+// Config defines the inputs to a truss service generation
+type Config struct {
+	// The first path in $GOPATH
+	GOPATH string
+
+	// The path to where .pb.go files protoc-gen-go creates will be written
```

Comment:
In this config it is the full path. You specify it on the command line with the go package name though. 

It is this way because I was passing this value to protoc to inform it where to write the .pb.go files. 

Though thinking about it because I already have the go path I can just `filepath.Join(cfg.GOPATH, "src", cfg.PBPackage)` to get the full path for protoc.